addons/mail/static/src/core/common/message.xml

```diff
@@ -96,7 +96,17 @@
                                                         <button t-if="message.type === 'email'" class="o-mail-Message-originalEmailText d-block float-end p-1 mb-1 btn btn-link opacity-75 fst-italic" t-att-class="env.inChatWindow and isAlignedRight ? 'me-3' : 'ms-3'" t-esc="originalEmailText" t-on-click="() => this.state.originalEmail = !this.state.originalEmail"/>
                                                         <em t-if="message.subject and !message.isSubjectSimilarToOriginThreadName and !message.isSubjectDefault" class="mb-1 me-2">Subject: <t t-esc="message.subject"/></em>
                                                         <div t-if="message.type === 'email'" t-ref="shadowBody"/>
+                                                        <t t-elif="message.showTranslation" t-out="message.translationValue"/>
                                                         <t t-elif="message.body" t-out="message.body"/>
+                                                        <small class="fst-italic text-muted">
+                                                            <p t-if="message.showTranslation">
+                                                                (Translated from: <t t-if="message.translationSource" t-out="message.translationSource"/>)
```

Comment:
So t-esc for plain text and t-out for markup when working on the client side right ?

Reply:
exactly 👌

Reply:
Functionally, `t-out` without `markup()` gives the same result as `t-esc`. `t-esc` is more performant... but that's only it. In general it's best to not optimise early.

Reply:
Perhaps my wording was misleading, I'm sorry. Performance isn't really at stake here, my point was mainly to provide additional context so that an informed choice could be made. It's true that I value consistency though, and would prefer not to mix `t-esc` and `t-out`